net/chrony/src/opnsense/mvc/app/controllers/OPNsense/Chrony/forms/general.xml

```diff
@@ -11,6 +11,12 @@
         <type>text</type>
         <help>Set the port chrony listen to.</help>
     </field>
+    <field>
+        <id>general.ntsclient</id>
+        <label>NTS Client Support</label>
+        <type>checkbox</type>
+        <help>Enable NTS in client mode. This will add another layer of security for peers when OPNsense is the client. Every server in Peers has to support NTS.</help>
```

Comment:
I'm not sure if this is the best approach. It will probably get in the way later. But I'm not strictly opposed to this.

Reply:
I know it would be better to use a grid but this plugin is more a special solution for ppl with special needs, rather than a replacement for core ntp service so I want to keep it small. 

Also when someone demands on NTS the intention should be that every server is offering it.

Reply:
ok, fine 👍